.github/aur/flux-bin/PKGBUILD.template

```diff
@@ -1,35 +1,29 @@
 # Maintainer: Aurel Canciu <aurelcanciu@gmail.com>
 # Maintainer: Hidde Beydals <hello@hidde.co>
 
-pkgname=flux-bin
-pkgver=${PKGVER}
-pkgrel=${PKGREL}
 _srcname=flux
 _srcver=${VERSION}
+pkgname=flux-bin
+pkgver=${_srcver//-/}
```

Comment:
As this is a template making use of `envsubst` for variable substitution, I am not quite sure this will actually work as intended in the produced `PKGBUILD` end result.

Reply:
As to Hidde's comment: I tested it locally and it works.

Reply:
Let's revert this line and use `PKGVER` instead.

Reply:
I've reverted